src/lib/filterTasksByClientClassification.ts

```diff
@@ -0,0 +1,24 @@
+export default function filterTasksByClientClassification(
+  tasks: TaskData[],
+  client: Client
+): TaskData[] {
+  return tasks.filter(
+    ({ classification }): boolean => {
+      return Object.keys(classification).reduce<boolean>(
+        (res, key): boolean => {
```

Comment:
`res` is an odd name for this variable. What does this mean? I usually use `accumulator` for a reduce

Reply:
I personally use `res` to mean "eventual result", but have changed to `acc` to be more clear of intentions. :+1: